### ISSUE TITLE
the remote folder by specific port

### ISSUE THREAD

**Issue by chuangchuangtan** [getcursor/cursor#38]
Hi.
I cannot access the remote folder by specific port, eg. ssh -i {mypemfile} root@{ip} -p 25785. I can login the machine in terminal.

**Comment by truell20** [comment]
Sorry about this! Do you have any logs in `~/Library/Logs/Cursor/main.log`?

**Comment by stephsix** [comment]
I cannot access some of my remote folders as well.
The log of a fail access as follows

```bash
01:22:59.549 › run command
01:22:59.550 › got out
01:22:59.551 › running
01:22:59.551 › running
01:22:59.552 › done
01:23:09.884 › ]]]]]]]]]]]]]] data  hello
cursordone
```
and nothing happened after that, with "Connect to SSH directory" dialog visiable.

Some of my remote folders can access in the same time, and the log as follows

```bash
01:23:18.541 › run command
01:23:18.543 › got out
01:23:18.543 › running
01:23:18.543 › running
01:23:18.544 › done
01:23:18.841 › ]]]]]]]]]]]]]] data  authz success

01:23:18.921 › ]]]]]]]]]]]]]] data  hello
cursordone

01:23:18.974 › run command
01:23:18.974 › got out
01:23:18.975 › running
01:23:18.975 › running
01:23:18.976 › done
01:23:18.997 › ]]]]]]]]]]]]]] data  total 16
drwxr-xr-x  4 root root 4096 Oct 16  2020 .
dr-xr-xr-x 19 root root 4096 Mar 18 01:23 ..
drwxr-xr-x  3 root root 4096 Oct 16  2020 git
drwxr-xr-x  4 root root 4096 Oct 16  2020 tmp
cursordone
```

I guess the problem may comes from the second command to fetch remote folder infos.

**Comment by danperks** [comment]
Was solved in v0.1.x with an OpenSSL update, and will likely not occur in v0.2.0.
